packages/design-system/src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,69 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.link {
+	font: tokens.$coral-paragraph-m;
+	color: tokens.$coral-color-accent-text;
+	background: none;
+	border: none;
+	border-bottom-color: currentColor;
+	cursor: pointer;
+
+	&,
```

Comment:
```suggestion
```
Means nothing in CSS modules

Reply:
Yes, it does

Reply:
It does?! I've got to see how it compiles

Reply:
It should be if we do not load Bootstrap v3 at first 😇 
It's a bullet-proof solution

Reply:
For info, here is the result if we don't have this block
![image](https://user-images.githubusercontent.com/10761073/149143474-40a45c3f-bbd2-400d-880d-386579b62e2c.png)

Bootstrap is loaded and takes precedence because it specifies a `hover` rule.